checkov/common/bridgecrew/integration_features/features/fixes_integration.py

```diff
@@ -78,6 +79,8 @@ def _get_fixes_for_file(self, filename, file_contents, failed_checks):
             error_message = extract_error_message(response)
             raise Exception(f'Get fixes request failed with response code {response.status_code}: {error_message}')
 
+        logging.debug(f'Response from fixes API: {response.content}')
+
         fixes = json.loads(response.content)
         return fixes[0]
```

Comment:
add protection to fixes - it would even be better.

Reply:
Done

Reply:
Oops, now done